### DISCUSSION TITLE
NPCs Names Distributor

### DISCUSSION THREAD

**Discussion by treaka** [alandtse/skyrim_vr_address_library#56] [Ideas]
This is a mod by [sasnikol](https://www.nexusmods.com/skyrimspecialedition/users/4613231), who also helps with the development for SPID. 

This mod was compatible with VR up to version 1.1.1. However 2.0 onwards requires a SKSE DLL and no VR conversion has been made. From comments sasnikol seems open to a VR version. Permissions for a conversion seem to be provided.

Nexus Link: [NPC Names Distributor](https://www.nexusmods.com/skyrimspecialedition/mods/73081?tab=description)
Source: [Github](https://github.com/adya/NPCs-Names-Distributor)

Thank you for allowing consideration of conversions.

**Comment by FlayaN** [comment]
Just randomly found this discussion now, I ported this to VR back in June of 2024 https://www.nexusmods.com/skyrimspecialedition/mods/121921
